### ISSUE TITLE
Using disable-log-cache.yml gives an error

### ISSUE THREAD

**Issue by gordcorp** [cloudfoundry/cf-deployment#602]
### What is this issue about?

The disable-log-cache.yml ops file is not removing a scheduler job which uses the log-cache release.

I think it's just missing:

```
- type: remove
  path: /instance_groups/name=scheduler/jobs/name=log-cache-expvar-forwarder
```

### What version of [cf-deployment](https://github.com/cloudfoundry/cf-deployment/releases) are you using?

cf-deployment v3.5.0

### Please include the `bosh deploy...` command, including all the operations files:
```
bosh -e main -d cf -n deploy cf-deployment/cf-deployment.yml \
    -o cf-deployment/operations/use-compiled-releases.yml \
    -o cf-deployment/operations/rename-network-and-deployment.yml \
    -o cf-deployment/operations/workaround/undo-metron-add-on.yml \
    -o cf-deployment/operations/use-external-dbs.yml \
    -o cf-deployment/operations/aws.yml \
    -o cf-deployment/operations/disable-router-tls-termination.yml \
    -o cf-deployment/operations/override-app-domains.yml \
    -o cf-deployment/operations/use-external-blobstore.yml \
    -o cf-deployment/operations/use-s3-blobstore.yml \
    -o cf-deployment/operations/stop-skipping-tls-validation.yml \
    -o cf-deployment/operations/disable-log-cache.yml
... -v ...
```

### Please provide output that helps describe the issue:

```
...
Task 210411 | 03:08:17 | Processing 6 existing jobs: Processing 6 existing jobs (00:00:00)
Task 210411 | 03:08:17 | Compiled Release has been created: silk/2.14.0 (00:00:00)
Task 210412 | 03:08:20 | Preparing deployment: Preparing deployment (00:00:04)
                      L Error: Job 'log-cache-expvar-forwarder' (instance group 'scheduler') references an unknown release 'log-cache'
Task 210412 | 03:08:24 | Error: Job 'log-cache-expvar-forwarder' (instance group 'scheduler') references an unknown release 'log-cache'
Updating deployment:
  Expected task '210412' to succeed but state is 'error'

Exit code 1

````
### What IaaS is this issue occurring on?

AWS

**Comment by heyjcollins** [comment]
This issue has been resolved in cf-deployment v3.4.0.`
I'm going to close this issue, but please re-open if the issue isn't resolved.

**Comment by heyjcollins** [comment]
@gordcorp - Adam's correct. I'm going to close this issue. If for some reason you're still experiencing this issue, feel free to reopen it.

Thank you,

Josh
